src/azure-cli/azure/cli/command_modules/role/_help.py

```diff
@@ -389,14 +389,10 @@
     [managed identities](https://aka.ms/azadsp-managed-identities) if available to avoid the need to use credentials.
 
 
-    By default, this command assigns the 'Contributor' role to the service principal at the subscription scope.
-    To reduce your risk of a compromised service principal, use --skip-assignment to avoid creating a role assignment,
-    then assign a more specific role and narrow the scope to a resource or resource group.
+    By default, this command does not create role assignment for the service principal.
```

Comment:
```suggestion
    By default, this command does not assign any role to the service principal.
```

Reply:
Thanks for the suggestion. Personally I would prefer `role assignment` as this is a specific term, as in `az role assignment create`.

Reply:
Thanks for the suggestion. Personally I would prefer `role assignment` as this is a specific term, as in `az role assignment create`.

Reply:
On a second though, `assign any role to` is definitely much simpler to understand.